api/controllers/login.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(store) {
 
 			return res.success(token);
 		} catch (e) {
+			if (e.Error === 'LockError') {
+				res.set('Retry-After', e.retry);
+				return res.failure('To many attempts.', 429);
```

Comment:
The message should be changed to:

`'Login have been locked for a short cooldown period due to failed login attempts.'`